common/install_azure_persistent_rdma_naming_monitor.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -ex
+
+#
+# setup systemd service
+#
+
+cat <<EOF >/usr/sbin/azure_persistent_rdma_naming_monitor.sh
+#!/bin/bash
+
+# monitoring service to check that hca_id's are named correctly
+# if incorrect, restart azure_persistent_rdma_naming.service
+
+while true; do 
+
+    for device in \$(ibdev2netdev -v | sort -n | cut -f2 -d' '); do
+        
+        link_layer=\$(ibv_devinfo -d \$device | sed -n 's/^[\ \t]*link_layer:[\ \t]*\([a-zA-Z]*\)\$/\1/p')
+
+        if [[ \$device != *"an"* && \$device != *"ib"* ]]; then 
+            sudo systemctl enable azure_persistent_rdma_naming.service
```

Comment:
do we need to run this as root here? and can we only enable when it's disabled?

Reply:
We can enable it if it's enabled or disabled -> if enabled already, the command does not fail. 
We can run it without sudo. From the command line it requires sudo, but in the service it does not - I will remove this.